deployment/dynatrace-azure-logs.sh

```diff
@@ -240,7 +228,7 @@ then
     check_arg --deployment-name "$DEPLOYMENT_NAME" "$DEPLOYMENT_NAME_REGEX"
     check_arg --resource-group "$RESOURCE_GROUP" ""
     check_arg --event-hub-connection-string "$EVENT_HUB_CONNECTION_STRING" "$EVENT_HUB_CONNECTION_STRING_REGEX"
-    check_arg --event-hub-name "$EVENT_HUB_NAME" "$EVENT_HUB_NAME_REGEX"
+    EVENT_HUB_NAME=$(echo "$EVENT_HUB_CONNECTION_STRING" | awk -F ';EntityPath=' '{print $2}')
```

Comment:
we still should run check_arg on extracted EVENT_HUB_NAME

Reply:
I'm not sure about that - our regex was incorrect (don't know if something changed in azure naming or we just used wrong regex) e.g. '_' is also correct in event hub name - imo checking if event hub connection string has EntityPath is enough

Reply:
I'm afraid of potential user errors, e.g. copying valid connection string with space at the end.
we have following options:
- still use regex, but maybe just print WARNING instead of finishing installation?
- try to extract also eventhub namespace and use Azure cli to validate that EH exists? print WARNING if it does not exists or no permission to list EH
- or at least trim whitespaces from connection string
Please decide on feasible option

Reply:
After discussion on zoom we decided not to complicate interactive mode (method for checking if event hub exists using azure cli), updating EVENT_HUB_CONNECTION_STRING regex will be enough (it will not accept whitespaces)